### PR TITLE
:bug: Fix doesn't work nameOf*

### LINKED ISSUE #750: nameOf* other than nameOfNbtCompoundTagKeys does not work
image

### PR DESCRIPTION
**Pull request by ChenCMD**
Fix #750
image

### CODE REVIEW

File: src/parsers/TeamArgumentParser.ts [hunk 1]
```diff
@@ -50,6 +52,18 @@ export class TeamArgumentParser extends ArgumentParser<string> {
             ))
         } else {
             if (this.isDefinition) {
```
Comment (SPGoding) [inline src/parsers/TeamArgumentParser.ts:54]: Same here.

File: src/parsers/ObjectiveArgumentParser.ts [hunk 2]
```diff
@@ -50,6 +52,18 @@ export class ObjectiveArgumentParser extends ArgumentParser<string> {
             ))
         } else {
             if (this.isDefinition) {
```
Comment (SPGoding) [inline src/parsers/ObjectiveArgumentParser.ts:54]: Any reason why the check is limited to definitions?

**Comment by ChenCMD** [comment]
I thought everyone would define it and then use the scoreboard and teams, which in hindsight is a silly idea.

**Review by SPGoding — APPROVED** [review]

**Pull request merged**